src/main/resources/com/arcbees/bourseje/client/resources/css/pages/candidates.gss

```diff
@@ -0,0 +1,42 @@
+.text_intro {
+    margin-bottom: 100px;
+}
+
+.text_intro h2,
```

Comment:
In this example, you could also add a class to your h2, called *text_intro_title* which would be even faster